Makefile

```diff
@@ -77,6 +90,7 @@ test-e2e-with-kafka-testonly:
 
 test-e2e-with-kafka:
 	UNINSTALL_STRIMZI="false" ./hack/strimzi.sh
+	./hack/tracing.sh
```

Comment:
Is this equivalent to `install-tracing-opentelemetry` here? Should we replace with that to make it more explicit?